src/App.tsx

```diff
@@ -21,7 +21,7 @@ const App = () => {
       <div className="App">
         <Switch>
           <Route path="/" exact>
-            <ResponsiveCards />
```

Comment:
Molecule can't really be a route entry point, that's why we have module subfolders